include/mbgl/map/map.hpp

```diff
@@ -182,7 +182,7 @@ class Map : private util::noncopyable {
 
     // Light
     void setLight(std::unique_ptr<style::Light>);
-    style::Light* getLight();
+    std::unique_ptr<style::Light> getLight();
```

Comment:
This should continue to return `style::Light*`; returning a `std::unique_ptr` gives up ownership, which we don't want to do.